tests/countries/test_libya.py

```diff
@@ -0,0 +1,157 @@
+#  holidays
+#  --------
+#  A fast, efficient Python library for generating country, province and state
+#  specific sets of holidays on the fly. It aims to make determining whether a
+#  specific date is a holiday as fast and flexible as possible.
+#
+#  Authors: Vacanza Team and individual contributors (see CONTRIBUTORS file)
+#           dr-prodigy <dr.prodigy.github@gmail.com> (c) 2017-2023
+#           ryanss <ryanssdev@icloud.com> (c) 2014-2017
+#  Website: https://github.com/vacanza/holidays
+#  License: MIT (see LICENSE file)
+
+from unittest import TestCase
+
+from holidays.countries.libya import Libya, LY, LBY
+from tests.common import CommonCountryTests
+
+
+class TestLibya(CommonCountryTests, TestCase):
+    @classmethod
+    def setUpClass(cls):
+        years = range(1968, 2050)
```

Comment:
Our current `start_year` is 2012
```suggestion
        years = range(2012, 2050)
```